core/shared/src/test/scala/com/monovore/decline/HelpSpec.scala

```diff
@@ -39,6 +53,10 @@ class HelpSpec extends AnyWordSpec with Matchers {
           |        Second option.
           |    --third <integer>
           |        Third option.
+          |    --flagOpt[=<string>]
```

Comment:
Let's give one of these two new options a short form, to see how that renders here.

(Looking at the code, I suspect we'll get `-F[=<metavar>]`, where the correct rendering would be `-F[<metavar>]`, in which case there's a minor bug to fix. But regardless the test will be useful to prevent any future trouble!)

Reply:
Hmm, let me check...

Reply:
I think it would also make sense to extend the UsageSpec.

Reply:
Just checked, you were right. Updated the implementation and tests in https://github.com/bkirwi/decline/pull/415/commits/fd75004574383159275a8a2179a05e7564fd94a2.